scripts/apps/archive/directives/MediaPreview.ts

```diff
@@ -115,6 +115,17 @@ export function MediaPreview(api, $rootScope, desks, superdesk, content, storage
             scope.$on('$destroy', () => {
                 elem.off('dragstart');
             });
+
+            scope.checkDuplicateKeywords = function() {
```

Comment:
not sure about the naming and the usage later being it a function, what about something like:
```ts
scope.uniqueKeywords = scope.selected.preview.keywords.filter((keyword) => ...);
```